Loop/View Controllers/InsulinDeliveryTableViewController.swift

```diff
@@ -2,13 +2,571 @@
 //  InsulinDeliveryTableViewController.swift
 //  Naterade
 //
-//  Created by Nathan Racklyeft on 3/11/16.
+//  Created by Nathan Racklyeft on 1/30/16.
```

Comment:
Moved this from LoopKit because we need to add the DeviceManager into it. I basically added a section to view logged insulin doses, and a "+" button in the top right to open the dose logging screen